sphinx_reredirects/__init__.py

```diff
@@ -62,6 +64,9 @@ def grab_redirects(self) -> Mapping[str, str]:
 
         # For each source-target redirect pair in conf.py
         for source, target in self.redirects_option.items():
+            # relativize target path
```

Comment:
If we rename the helper function, it will be just one line and easier to understand.
```suggestion
            target = self._relativize_target_path(source, target)
```